ideascube/mediacenter/forms.py

```diff
@@ -17,6 +18,9 @@ class Meta:
         widgets = {
             'lang': LangSelect,
         }
+        help_texts = {
+            'original': _('Maximum size : 500 Mo.'),
+        }
```

Comment:
It's not cause django's people does strange things that we have to do the same ;)

This `help_texts` is only useful in this specific form (upload form), not everywhere.
`help_texts` is a "valid" way to set the help text only for one form.

However, there is no other form using this model, so it is not so important for now. I will not argue too long on this. If you prefer to have the `help_text` in the model, I will do it. Make as every one has advantage also :)